### ISSUE TITLE
[🐛 Bug]: test/unit/selenium/webdriver/ie/test_ie_options.py::test_element_scroll_behavior fails w/ selenium-python 4.11.0

### ISSUE THREAD

**Issue by mgorny** [SeleniumHQ/selenium#12459]
### What happened?

With `selenium-4.11.0` and git 4beab57e3f17d6f34102050dc5994e1b74c94c6b, I can reproduce the following unit test failure:

```
FAILED test/unit/selenium/webdriver/ie/test_ie_options.py::test_element_scroll_behavior - ValueError: elementScrollBehavior should be of type int
```

FWICS, the problem is that `ElementScrollBehavior` is defined as an `Enum`, while an `int` is expected. The issue can possibly be fixed by changing it to an `IntEnum` instead.

The other possibility is to change `element_scroll_behavior` to expect `Enum` but that requires more changes to the code and probably breaks backwards compatibility.

### How can we reproduce the issue?

```shell
pip install . pytest
python -m pytest -vvl test/unit
```

### Relevant log output

```shell
============================================================== FAILURES ===============================================================
____________________________________________________ test_element_scroll_behavior _____________________________________________________

opts = <selenium.webdriver.ie.options.Options object at 0x7fbde0ee1890>

    def test_element_scroll_behavior(opts):
>       opts.element_scroll_behavior = ElementScrollBehavior.BOTTOM

opts       = <selenium.webdriver.ie.options.Options object at 0x7fbde0ee1890>

test/unit/selenium/webdriver/ie/test_ie_options.py:53: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <selenium.webdriver.ie.options._IeOptionsDescriptor object at 0x7fbde1100f90>
obj = <selenium.webdriver.ie.options.Options object at 0x7fbde0ee1890>, value = <ElementScrollBehavior.BOTTOM: 1>

    def __set__(self, obj, value) -> None:
        if not isinstance(value, self.expected_type):
>           raise ValueError(f"{self.name} should be of type {self.expected_type.__name__}")
E           ValueError: elementScrollBehavior should be of type int

obj        = <selenium.webdriver.ie.options.Options object at 0x7fbde0ee1890>
self       = <selenium.webdriver.ie.options._IeOptionsDescriptor object at 0x7fbde1100f90>
value      = <ElementScrollBehavior.BOTTOM: 1>

selenium/webdriver/ie/options.py:59: ValueError
```

### Operating System

Gentoo Linux

### Selenium version

Python 4.11.0

### What are the browser(s) and version(s) where you see this issue?

n/a

### What are the browser driver(s) and version(s) where you see this issue?

n/a

**Comment by titusfortner** [comment]
@sandeepsuryaprasad it looks like the code isn't getting to the lines it needs here? Do we need to move the conditional up?

**Comment by sandeepsuryaprasad** [comment]
@titusfortner I will check.

**Comment by sandeepsuryaprasad** [comment]
> @sandeepsuryaprasad it looks like the code isn't getting to the lines it needs here? Do we need to move the conditional up?

@titusfortner I have fixed this. It is my mistake that I was getting the `int` value of `Enum` object in `ElementScrollBehavior.BOTTOM` and `ElementScrollBehavior.TOP`. Now I have fixed it by removing `.value`

**Comment by diemol** [comment]
Merged the PR that fixes this, most likely 4.11.2 is coming soon.
